### DISCUSSION TITLE
retype input files location?  (github pages with retype)

### DISCUSSION THREAD

**Discussion by benkoshy** [retypeapp/retype#462] [Q&A] [answered]
Hi there,

I am successfully using retype via Githubs actions.  see here: 

The **input files** are currently contained within the `docs` section of my repository. .

* Suppose I want the **input files** to be in the `retype` folder - will setting the directory in the `retype.yml` to `retype` allow this to happen - I mean all the input files `*.md` etc.?
* In other words, how does the retype action know where the retype files are contained?

Any pointers would be much appreciated.

Thank you!

**Comment by geoffreymcgill** [comment, marked as answer]
> will setting the directory in the retype.yml to retype allow this to happen - I mean all the input files *.md etc.?

Correct.

> In other words, how does the retype action know where the retype files are contained?

The [`input`](https://retype.com/configuration/project/#input) config instructs Retype on where to find the source `.md` files for your project.

Setting `input: retype` in your projects `retype.yml` file should be the only change required, although that path is relative to the location of the `retype.yml` file. If currently you have `input: docs` and would like to rename that `docs` folder to `retype`, then just revise to `input: retype`. 

Hope this helps.
